### PR TITLE
IBX-5710: Shipping REST API

### PR DESCRIPTION
**Pull request by MagdalenaZuba**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://issues.ibexa.co/browse/IBX-5710
| Versions      | 4.6
| Edition       | N/A

Added templates to:
- GET /shipments (ibexa/shipping#29)
- GET /shipments/{identifier}
- PATCH /shipments/{identifier}
- GET /shipping/methods (ibexa/shipping#25)
- GET /shipping/methods/{identifier}
- GET /shipping/method-types (ibexa/shipping#24)
- GET /shipping/method-types/{identifier}
- GET /orders/order/{identifier}/shipments (ibexa/shipping#29)

TODO:
- [x] Rebuild rest_api_reference.html one last time before merge

#### Checklist

- [ ] Text renders correctly
- [ ] Text has been checked with vale
- [ ] Description metadata is up to date
- [ ] Redirects cover removed/moved pages
- [ ] Code samples are working
- [ ] PHP code samples have been fixed with PHP CS fixer
- [ ] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: docs/api/rest_api/rest_api_reference/input/ez-types.raml [hunk 1]
```diff
@@ -3679,4 +3679,47 @@ PaymentMethodViewInput:
...
+Shipment:
+    description: 'This class represents a a single shipment.'
+    type: object
+    properties:
+        identifier:
+            type: string
+        status:
+            type: string
+        transitions:
+            type: TransitionList[]
```
Comment (adamwojs) [inline docs/api/rest_api/rest_api_reference/input/ez-types.raml:3693]: `TransitionList` definition is missing.

File: docs/api/rest_api/rest_api_reference/input/ez-orders.raml [hunk 2]
```diff
@@ -75,7 +75,33 @@ displayName: Orders
...
+                        description: The shipments schema encoded in XML or JSON format.
+                        example:  |
+                            application/vnd.ibexa.api.ShipmentRefList+xml
+                responses:
+                    200:
+                        description: OK - loads the shipments.
+                        body:
+                            application/vnd.ibexa.api.Shipment+xml:
+                                type: Shipment
+                                example: !include examples/shipments/GET/ShipmentRefList.xml.example
```
Comment (adamwojs) [inline docs/api/rest_api/rest_api_reference/input/ez-orders.raml:97]: Example which you refer to doesn't exists

File: docs/api/rest_api/rest_api_reference/input/ez-orders.raml [hunk 3]
```diff
@@ -75,7 +75,33 @@ displayName: Orders
...
+                            application/vnd.ibexa.api.Shipment+json:
+                                type: Shipment
+                                example: !include examples/shipments/GET/ShipmentRefList.json.example
```
Comment (adamwojs) [inline docs/api/rest_api/rest_api_reference/input/ez-orders.raml:100]: Example which you refer to doesn't exists

File: docs/api/rest_api/rest_api_reference/input/ez-orders.raml [hunk 4]
```diff
@@ -75,7 +75,33 @@ displayName: Orders
                     description: Error - the user has no permission to read orders.
                 404:
                     description: Error - the order does not exist.
-
+        /shipments:
```
Comment (adamwojs) [inline docs/api/rest_api/rest_api_reference/input/ez-orders.raml:78]: Are you sure is a right place for this endpoint ?

File: docs/api/rest_api/rest_api_reference/input/ez-shipments.raml [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+            description: The shipments schema encoded in XML or JSON format.
+            example:  |
+                application/vnd.ibexa.api.ShipmentRefList+xml
+    responses:
+        200:
+            description: OK - loads the shipments.
+            body:
+                application/vnd.ibexa.api.Shipment+xml:
+                    type: Shipment
+                    example: !include examples/shipments/GET/ShipmentRefList.xml.example
```
Comment (adamwojs) [inline docs/api/rest_api/rest_api_reference/input/ez-shipments.raml:19]: Example which you refer to doesn't exists

File: docs/api/rest_api/rest_api_reference/input/ez-shipments.raml [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+                application/vnd.ibexa.api.Shipment+json:
+                    type: Shipment
+                    example: !include examples/shipments/GET/ShipmentRefList.json.example
```
Comment (adamwojs) [inline docs/api/rest_api/rest_api_reference/input/ez-shipments.raml:22]: Example which you refer to doesn't exists

File: docs/api/rest_api/rest_api_reference/input/ez-shipments.raml [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+            Content-Type:
+                description: The ShipmentUpdate schema encoded in XML or JSON format.
+                example:  |
+                    application/vnd.ibexa.api.ShipmentUpdate+json
+        responses:
+            200:
+                body:
+                    application/vnd.ibexa.api.Shipment+json:
+                        type: ShipmentUpdate
+                        example: !include examples/shipments/identifier/PATCH/Shipment.xml.example
```
Comment (adamwojs) [inline docs/api/rest_api/rest_api_reference/input/ez-shipments.raml:77]: Example which you refer to doesn't exists

**Review by adamwojs — CHANGES_REQUESTED** [review]

File: docs/api/rest_api/rest_api_reference/input/ez-orders.raml (hunk 4 above)
Comment (MagdalenaZuba) [reply docs/api/rest_api/rest_api_reference/input/ez-orders.raml:78]: It's for GET /orders/order/{identifier}/shipments

**Comment by adamwojs** [comment]
@MagdalenaZuba rebase is needed here

**Comment by MagdalenaZuba** [comment]
@adamwojs this PR is marked as work in progress, I just provided skeleton, Andrew took over on Aug 31 and will provide examples and fill in missing information as well as perform rebase

**Comment by adamwojs** [comment]
I've convert it to a draft in order to skip going through this PR next time :wink:

File: docs/api/rest_api/rest_api_reference/input/examples/shipping/methods/identifier/GET/ShippingMethod.xml.example [hunk 8]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ShippingMethod media-type="application/vnd.ibexa.api.ShippingMethod+xml" href="/api/ibexa/v2/shipping/method-types/method1">
```
Comment (adriendupuis) [inline docs/api/rest_api/rest_api_reference/input/examples/shipping/methods/identifier/GET/ShippingMethod.xml.example:2]: There might be something wrong here.
Like in ShippingMethodRefList, it probably should be
```suggestion
<ShippingMethod media-type="application/vnd.ibexa.api.ShippingMethod+xml" href="/api/ibexa/v2/shipping/method/method1">
```

It comes from https://github.com/ibexa/shipping/pull/25/files#diff-4ef4d687a3bdb5cdbbb1ae7f0bf7ac0a45372a32d8def7958df51b9d60d2f07aR43

@alongosz will check there if this is intended or if there was a confusion.

File: docs/api/rest_api/rest_api_reference/input/examples/shipments/identifier/PATCH/ShipmentUpdate.json.example [hunk 9]
```diff
@@ -0,0 +1,14 @@
+{
+    "ShipmentUpdate": {
+        "identifier": "new_identifier",
+        "transition": "prepare",
+        "Owner": {
+            "_media-type": "application/vnd.ibexa.api.User+json",
+            "_href": "/api/ibexa/v2/user/users/14"
+        },
+        "Context": {
```
Comment (adriendupuis) [inline docs/api/rest_api/rest_api_reference/input/examples/shipments/identifier/PATCH/ShipmentUpdate.json.example:9]: TODO: How this is reflected on resulting …/[PATCH/Shipment.json.example](https://github.com/ibexa/documentation-developer/pull/2053/files#diff-a95c7ea0c13ef731ef50d859642e19d9e8356e4839076bcdcdd9090923d0185a)?

File: docs/api/rest_api/rest_api_reference/input/examples/shipping/methods/identifier/GET/ShippingMethod.xml.example (hunk 8 above)
Comment (adriendupuis) [reply docs/api/rest_api/rest_api_reference/input/examples/shipping/methods/identifier/GET/ShippingMethod.xml.example:2]: See ibexa/shipping#57

Comment (adriendupuis) [reply docs/api/rest_api/rest_api_reference/input/examples/shipping/methods/identifier/GET/ShippingMethod.xml.example]: - ibexa/shipping#57 merged
- Example fixed in 9d16685ea246f72ef870a9cc6bff08f9dbd8df2d

File: docs/api/rest_api/rest_api_reference/input/examples/shipments/identifier/PATCH/ShipmentUpdate.json.example (hunk 9 above)
Comment (adriendupuis) [reply docs/api/rest_api/rest_api_reference/input/examples/shipments/identifier/PATCH/ShipmentUpdate.json.example:9]: It seems there is no visitor to have a shipment context in this output.

File: docs/api/rest_api/rest_api_reference/input/examples/shipments/identifier/PATCH/Shipment.json.example [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+    },
+    "createdAt": "2023-06-26T15:40:28+02:00",
+    "updatedAt": "2023-06-26T15:40:28+02:00",
+    "Owner": {
+      "_media-type": "application\/vnd.ibexa.api.User+json",
+      "_href": "\/api\/ibexa\/v2\/user\/users\/14"
+    },
+    "Method": {
+      "_media-type": "application\/vnd.ibexa.api.ShippingMethod+json",
+      "_href": "\/api\/ibexa\/v2\/shipping\/methods\/inpost_parcel_locker_delivery"
```
Comment (alongosz) [inline docs/api/rest_api/rest_api_reference/input/examples/shipments/identifier/PATCH/Shipment.json.example:30]:
```suggestion
      "_href": "\/api\/ibexa\/v2\/shipping\/methods\/parcel_locker_delivery"
```

File: docs/api/rest_api/rest_api_reference/input/ez-shipping.raml [hunk 11]
```diff
@@ -0,0 +1,98 @@
...
+                    description: The shipping method schema encoded in XML or JSON format.
+                    example:  |
+                        application/vnd.ibexa.api.ShippingMethodType+json
+            responses:
+                200:
+                    description: OK - loads the shipping method.
+                    body:
+                        application/vnd.ibexa.api.ShippingMethodType+json:
+                            type: ShippingMethodType
+                            example: !include examples/shipping/method-types/identifier/GET/ShippingMethodType.json.example
```
Comment (alongosz) [inline docs/api/rest_api/rest_api_reference/input/ez-shipping.raml:94]: Seems we have XML example file, but it wasn't specified here:
```suggestion
                        application/vnd.ibexa.api.ShippingMethodType+xml:
                            type: ShippingMethodType
                            example: !include examples/shipping/method-types/identifier/GET/ShippingMethodType.xml.example
                        application/vnd.ibexa.api.ShippingMethodType+json:
                            type: ShippingMethodType
                            example: !include examples/shipping/method-types/identifier/GET/ShippingMethodType.json.example
```

File: docs/api/rest_api/rest_api_reference/input/examples/shipments/GET/ShipmentRefList.json.example [hunk 12]
```diff
@@ -0,0 +1,24 @@
...
+                "_media-type": "application\/vnd.ibexa.api.Shipment+json",
+                "_href": "\/api\/ibexa\/v2\/shipments\/baz"
+            },
+            {
+                "_media-type": "application\/vnd.ibexa.api.Shipment+json",
+                "_href": "\/api\/ibexa\/v2\/shipments\/qux"
+            }
+        ]
+    }
+}
```
[the remark this answers is not in the document]
Comment (adriendupuis) [reply docs/api/rest_api/rest_api_reference/input/examples/shipments/GET/ShipmentRefList.json.example]: Solved in 15d889d44a6728e602b8b4542341f1ee8adc072e

File: docs/api/rest_api/rest_api_reference/input/ez-shipments.raml [hunk 13]
```diff
@@ -0,0 +1,93 @@
+get:
```
[the remark this answers is not in the document]
Comment (adriendupuis) [reply docs/api/rest_api/rest_api_reference/input/ez-shipments.raml:3]: Fixed in 7bc6eae6b49c8ebaa37f7bc100e241f77659b335

File: docs/api/rest_api/rest_api_reference/input/ez-shipments.raml [hunk 14]
```diff
@@ -0,0 +1,93 @@
...
+                    example: !include examples/shipments/GET/ShipmentRefList.xml.example
+                application/vnd.ibexa.api.Shipment+json:
+                    type: Shipment
+                    example: !include examples/shipments/GET/ShipmentRefList.json.example
+        401:
+            description: Error - the user has no permission to read shipment list.
+        404:
+            description: Error - the shipment does not exist.
+
+/identifier:
```
[the remark this answers is not in the document]
Comment (adriendupuis) [reply docs/api/rest_api/rest_api_reference/input/ez-shipments.raml]: Fixed in 7bc6eae6b49c8ebaa37f7bc100e241f77659b335

File: docs/api/rest_api/rest_api_reference/input/ez.raml [hunk 15]
```diff
@@ -54,6 +54,10 @@ version: v3.3.2
 
 /payment-method: !include ez-payment-methods.raml
 
+/shipping: !include ez-shipping.raml
+
+/shipments: !include ez-shipments.raml
```
[the remark this answers is not in the document]
Comment (adriendupuis) [reply docs/api/rest_api/rest_api_reference/input/ez.raml:59]: I'm fully agree with that. I'll make a rebranding PR at some point. And, yes, this prefix thing is not very relevant. The only reason it was done this way I can guess today is to have ez.raml first in file list 😓 
We could remove those prefix and find a new name for ez.raml that won't become a base URL "namespace" in the future. main.raml, index.raml, rest_api_reference.raml,… Again, let's pick this new name in another PR.

File: docs/api/rest_api/rest_api_reference/input/examples/shipping/methods/GET/ShippingMethodRefList.json.example [hunk 16]
```diff
@@ -0,0 +1,16 @@
...
+        "_media-type": "application\/vnd.ibexa.api.ShippingMethodRefList+json",
+        "_href": "\/api\/ibexa\/v2\/shipping\/methods",
+        "ShippingMethodRefs": [
+            {
+                "_media-type": "application\/vnd.ibexa.api.ShippingMethod+json",
+                "_href": "\/api\/ibexa\/v2\/shipping\/methods\/parcel_delivery"
+            },
+            {
+                "_media-type": "application\/vnd.ibexa.api.ShippingMethod+json",
+                "_href": "\/api\/ibexa\/v2\/shipping\/methods\/inpost_parcel_locker"
```
[the remark this answers is not in the document]
Comment (adriendupuis) [reply docs/api/rest_api/rest_api_reference/input/examples/shipping/methods/GET/ShippingMethodRefList.json.example]: Done in 63a3945e029aa0ca0c267f22eb073af9140cc761 and/or 5e24f8fddefdd2bd8e0c4e068cb1ca809f9e8fbf

File: docs/api/rest_api/rest_api_reference/input/examples/shipments/identifier/PATCH/Shipment.json.example (hunk 10 above)
Comment (adriendupuis) [reply docs/api/rest_api/rest_api_reference/input/examples/shipments/identifier/PATCH/Shipment.json.example]: Done in 63a3945e029aa0ca0c267f22eb073af9140cc761 and/or 5e24f8fddefdd2bd8e0c4e068cb1ca809f9e8fbf

File: docs/api/rest_api/rest_api_reference/input/ez-types.raml [hunk 17]
```diff
@@ -3688,9 +3688,75 @@ PaymentMethodViewInput:
...
+        Method:
+            type: ShippingMethod
+
+ShipmentUpdate:
+    description: 'This class represents the update of a shipment.'
+    #TODO
+
+ShippingMethod:
+    description: 'This class represents a shipping method.'
+    #TODO
```
[the remark this answers is not in the document]
Comment (adriendupuis) [reply docs/api/rest_api/rest_api_reference/input/ez-types.raml:3762]:
```suggestion
    type: object
    properties:
        identifier:
            type: string
        name:
            type: string
        names:
            type: object
            properties:
                value:
                    type: MultilingualValue[]
        description:
            type: string
        descriptions:
            type: object
            properties:
                value:
                    type: MultilingualValue[]
        enabled:
            type: boolean
        options:
            type: object
            properties:
                value:
                    type: KeyValue[]
        regions:
            type: Region[]
        type:
            type: ShippingMethodType[]

MultilingualValue:
    description: 'Object that represents a multilingual (translated) value.'
    type: object
    properties:
        _languageCode:
            description: 'Language code.'
            type: string
        '#text':
            description: 'Translation contents.'
            type: string

KeyValue:
    description: 'Key-value structure'
    type: object
    properties:
        _key:
            type: string
        '#text':
            type: string            
```

**Review by alongosz — APPROVED** [review]

File: docs/api/rest_api/rest_api_reference/input/ez-shipping.raml [hunk 18]
```diff
@@ -0,0 +1,101 @@
+displayName: Shipping methods
+
+/methods:
+    get:
+        displayName: Load shipping methods
+        description: Loads shipping methods.
+        headers:
+            Accept:
+                description: Shipping method reference - If set, the shipping method returned in XML or JSON format.
```
Comment (dabrt) [inline docs/api/rest_api/rest_api_reference/input/ez-shipping.raml:9]:
```suggestion
                description: Shipping method reference - if set, the shipping method is returned in XML or JSON format.
```

File: docs/api/rest_api/rest_api_reference/input/ez-shipping.raml [hunk 19]
```diff
@@ -0,0 +1,101 @@
...
+                        example: !include examples/shipping/methods/GET/ShippingMethodRefList.json.example
+            401:
+                description: Error - the user has no permission to read shipping methods.
+    /{identifier}:
+        get:
+            displayName: Load shipping method based on its identifier
+            description: Loads shipping method based on its identifier.
+            headers:
+                Accept:
+                    description: Shipping method - If set, the shipping method returned in XML or JSON format.
```
Comment (dabrt) [inline docs/api/rest_api/rest_api_reference/input/ez-shipping.raml:31]:
```suggestion
                    description: Shipping method - if set, the shipping method is returned in XML or JSON format.
```

File: docs/api/rest_api/rest_api_reference/input/ez-shipping.raml [hunk 20]
```diff
@@ -0,0 +1,101 @@
...
+                404:
+                    description: Error - the shipping method does not exist.
+
+/method-types:
+    get:
+        displayName: Load shipping methods types
+        description: Loads shipping methods types.
+        headers:
+            Accept:
+                description: Shipping method type reference - If set, the shipping method returned in XML or JSON format.
```
Comment (dabrt) [inline docs/api/rest_api/rest_api_reference/input/ez-shipping.raml:56]:
```suggestion
                description: Shipping method type reference - if set, the shipping method is returned in XML or JSON format.
```

File: docs/api/rest_api/rest_api_reference/input/ez-shipping.raml [hunk 21]
```diff
@@ -0,0 +1,101 @@
...
+            404:
+                description: Error - the shipping method type does not exist.
+
+    /{identifier}:
+        get:
+            displayName: Load shipping method type based on its identifier
+            description: Loads shipping methods type based on their identifiers.
+            headers:
+                Accept:
+                    description: Shipping method type - If set, the shipping method returned in XML or JSON format.
```
Comment (dabrt) [inline docs/api/rest_api/rest_api_reference/input/ez-shipping.raml:81]:
```suggestion
                    description: Shipping method type - if set, the shipping method is returned in XML or JSON format.
```

File: docs/api/rest_api/rest_api_reference/input/ez-types.raml [hunk 22]
```diff
@@ -142,7 +142,7 @@ ContentCreateStruct:
             required: true
             type: string
         modificationDate:
-            descritpion: 'Modification date. If not given the current integer is used.'
+            description: 'Modification date. If not given the current integer is used.'
```
Comment (dabrt) [inline docs/api/rest_api/rest_api_reference/input/ez-types.raml:145]:
```suggestion
            description: 'Modification date. If not given, the current integer is used.'
```

File: docs/api/rest_api/rest_api_reference/input/ez-types.raml [hunk 23]
```diff
@@ -3719,9 +3719,132 @@ PaymentMethodViewInput:
...
+            type: integer
+        identifier:
+            type: string
+        status:
+            type: string
+        transitions:
+            type: object
+            properties:
+                transitionList:
+                    description: 'Available next steps in the shipment workflow for the current status'
```
Comment (dabrt) [inline docs/api/rest_api/rest_api_reference/input/ez-types.raml:3736]:
```suggestion
                    description: 'Next steps available in the shipment workflow for the current status'
```

File: docs/api/rest_api/rest_api_reference/input/ez-orders.raml [hunk 24]
```diff
@@ -75,6 +75,35 @@ displayName: Orders
...
+                        description: OK - loads the shipments.
+                        body:
+                            application/vnd.ibexa.api.Shipment+xml:
+                                type: Shipment
+                                example: !include examples/shipments/GET/ShipmentRefList.xml.example
+                            application/vnd.ibexa.api.Shipment+json:
+                                type: Shipment
+                                example: !include examples/shipments/GET/ShipmentRefList.json.example
+                    401:
+                        description: Error - the user has no permission to read shipment list.
```
Comment (dabrt) [inline docs/api/rest_api/rest_api_reference/input/ez-orders.raml:103]:
```suggestion
                        description: Error - the user has no permission to read shipment lists.
```

File: docs/api/rest_api/rest_api_reference/input/ez-shipments.raml [hunk 25]
```diff
@@ -0,0 +1,95 @@
...
+            description: OK - loads the shipments.
+            body:
+                application/vnd.ibexa.api.Shipment+xml:
+                    type: Shipment
+                    example: !include examples/shipments/GET/ShipmentRefList.xml.example
+                application/vnd.ibexa.api.Shipment+json:
+                    type: Shipment
+                    example: !include examples/shipments/GET/ShipmentRefList.json.example
+        401:
+            description: Error - the user has no permission to read shipment list.
```
Comment (dabrt) [inline docs/api/rest_api/rest_api_reference/input/ez-shipments.raml:26]:
```suggestion
            description: Error - the user has no permission to read shipment lists.
```

File: docs/api/rest_api/rest_api_reference/input/ez-shipments.raml [hunk 26]
```diff
@@ -0,0 +1,95 @@
...
+                description: OK - loads the shipment.
+                body:
+                    application/vnd.ibexa.api.Shipment+xml:
+                        type: Shipment
+                        example: !include examples/shipments/identifier/GET/Shipment.xml.example
+                    application/vnd.ibexa.api.Shipment+json:
+                        type: Shipment
+                        example: !include examples/shipments/identifier/GET/Shipment.json.example
+            401:
+                description: Error - the user has no permission to read shipment.
```
Comment (dabrt) [inline docs/api/rest_api/rest_api_reference/input/ez-shipments.raml:54]:
```suggestion
                description: Error - the user has no permission to read shipments.
```

File: docs/api/rest_api/rest_api_reference/input/ez-shipping.raml [hunk 27]
```diff
@@ -0,0 +1,101 @@
...
+                    description: OK - loads the shipping method.
+                    body:
+                        application/vnd.ibexa.api.ShippingMethod+xml:
+                            type: ShippingMethod
+                            example: !include examples/shipping/methods/identifier/GET/ShippingMethod.xml.example
+                        application/vnd.ibexa.api.ShippingMethod+json:
+                            type: ShippingMethod
+                            example: !include examples/shipping/methods/identifier/GET/ShippingMethod.json.example
+                401:
+                    description: Error - the user has no permission to read shipping method.
```
Comment (dabrt) [inline docs/api/rest_api/rest_api_reference/input/ez-shipping.raml:46]:
```suggestion
                    description: Error - the user has no permission to read shipping methods.
```

File: docs/api/rest_api/rest_api_reference/input/ez-shipping.raml [hunk 28]
```diff
@@ -0,0 +1,101 @@
...
+                description: OK - loads the shipping method.
+                body:
+                    application/vnd.ibexa.api.ShippingMethodTypesList+xml:
+                        type: ShippingMethodTypesList
+                        example: !include examples/shipping/method-types/GET/ShippingMethodTypesList.xml.example
+                    application/vnd.ibexa.api.ShippingMethodTypesList+json:
+                        type: ShippingMethodTypesList
+                        example: !include examples/shipping/method-types/GET/ShippingMethodTypesList.json.example
+            401:
+                description: Error - the user has no permission to read shipping methods type.
```
Comment (dabrt) [inline docs/api/rest_api/rest_api_reference/input/ez-shipping.raml:71]:
```suggestion
                description: Error - the user has no permission to read shipping methods types.
```

File: docs/api/rest_api/rest_api_reference/input/ez-shipping.raml [hunk 29]
```diff
@@ -0,0 +1,101 @@
...
+                    description: OK - loads the shipping method.
+                    body:
+                        application/vnd.ibexa.api.ShippingMethodType+xml:
+                            type: ShippingMethodType
+                            example: !include examples/shipping/method-types/identifier/GET/ShippingMethodType.xml.example
+                        application/vnd.ibexa.api.ShippingMethodType+json:
+                            type: ShippingMethodType
+                            example: !include examples/shipping/method-types/identifier/GET/ShippingMethodType.json.example
+                401:
+                    description: Error - the user has no permission to read shipping method.
```
Comment (dabrt) [inline docs/api/rest_api/rest_api_reference/input/ez-shipping.raml:99]:
```suggestion
                    description: Error - the user has no permission to read shipping methods.
```

**Review by dabrt — CHANGES_REQUESTED** [review]

**Review by dabrt — APPROVED** [review]

**Comment by adriendupuis** [comment]
Probably ready to merge but wait for #2289 which will remove the pollution of unrelated entries.

**Review by adriendupuis — APPROVED** [review]

File: docs/api/rest_api/rest_api_reference/input/ez-orders.raml [hunk 30]
```diff
@@ -75,6 +75,35 @@ displayName: Orders
                     description: Error - the user has no permission to read orders.
                 404:
                     description: Error - the order does not exist.
+                    
+        /shipments:
+            get:
+                displayName: Load shipments
```
Comment (adriendupuis) [inline docs/api/rest_api/rest_api_reference/input/ez-orders.raml:81]: It looks wrong. According to https://github.com/ibexa/shipping/pull/40/files#diff-b50dddb91d44700cc0b64f88a159499eb019b780afc5ec10913f8ed0cd145eef this root is used with POST to create a shipment.

**Pull request merged**